platform/security/sts/security-sts-server/src/main/java/ddf/security/sts/SubjectDNConstraintsInterceptor.java

```diff
@@ -74,8 +75,8 @@ public void handleMessage(Message message) throws Fault {
 
     public Collection<Pattern> setSubjectDNPatterns(String subjectDNConstraints) {
         ArrayList<Pattern> subjectDNPatterns = new ArrayList<>();
-        if (subjectDNConstraints != null) {
-            String[] patterns = subjectDNConstraints.split(",");
+        if (StringUtils.isNotEmpty(subjectDNConstraints)) {
```

Comment:
❓ Assuming we make this method `private`, can we replace the check on L56 with this one and get rid of this `if` statement? Or only have the check here, return an empty `List` if the constraint is `null` or empty and have the calling code trigger on that instead of the `subjectDNContraints` value? Just trying to avoid double `subjectDNContraints` validations that could become out-of-sync in the future.